docs-chef-io/content/server/upgrades.md

```diff
@@ -46,6 +46,16 @@ Chef Infra Server 14.0 moved from Solr to Elasticsearch as its search index.
 
 The Chef Infra Server 14 upgrade does not automatically reindex existing external Elasticsearch installations.
 
+#### Upgrading to 14.13
+
+Chef Infra Server 14.13 supports External Opensearch for indexing. Migrating from Elasticsearch to External Opensearch needs ontime reindexing and reconfigure after upgrade. This process may take longer depending on your server hardware and the size of the node objects on your Chef Infra Server.
+
+{{< note >}}
+
+Set the `opensearch['external']` & `opensearch['external_url']` attribute in [chef-server.rb]({{< relref "config_rb_server_optional_settings" >}}) to enable external Opensearch for indexing.
```

Comment:
Since this setting is required, it should be presented as a step instead of in a note. Please add example values for the config settings.

Reply:
Updated as per the feedback. Please re-verify